testing/cert.go

```diff
@@ -9,20 +9,12 @@ import (
 	"crypto/x509"
 	"fmt"
 	"math/rand"
-	"time"
+	_ "time"
 
 	gitjujutesting "github.com/juju/testing"
 	utilscert "github.com/juju/utils/cert"
-
-	"github.com/juju/juju/cert"
 )
 
-func init() {
```

Comment:
Can verifyCertificates go?

Reply:
Yep the validation is done on creation. It's a machine created file.